tests/automated_install.sh

```diff
@@ -132,7 +132,7 @@ vagrant ssh -c "cd chef-bcpc; source proxy_setup.sh; ./wait-for-hosts.sh ${VM_LI
 snapshotVMs "${SNAP_POST_PXE}"
 
 printf "#### Chef the nodes with Basic role\n"
-vagrant ssh -c "cd chef-bcpc; ./cluster-assign-roles.sh $BACH_ENVIRONMENT Basic"
+vagrant ssh -c "cd chef-bcpc; sudo ./cluster-assign-roles.sh $BACH_ENVIRONMENT Basic"
```

Comment:
we just removed the need to `sudo` all bootstrap-related commands because it considered a bug. why did we put it back?

Reply:
Oh awesome! I figured we still had the Gem-hell and was not trying to solve that here. If the Gem-hell's fixed we can chmod the key and all should be good. I'll see where things are at.